standalone/index.html

```diff
@@ -224,31 +290,31 @@
         width: calc(100% - 6px);
         border-width: 0 0px 0 1px;
         border-style: solid;
-        border-color: gray;
-        background: white;
+        border-color: var(--border-color);
+        background: var(--testcase-logs-bg-color);
       }
       .testcaselog {
         display: flex;
       }
       .testcaselog:nth-child(odd) {
-        background: #fff;
+        background: var(--testcase-log-odd-bg-color);
       }
       .testcaselog:nth-child(even) {
-        background: #f8f8f8;
+        background: var(--testcase-log-even-bg-color);
       }
-      .testcaselogbtn {
-        background-image: url(data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAwAAAAMAQMAAABsu86kAAAABlBMVEUAAAAAAAClZ7nPAAAAAXRSTlMAQObYZgAAACRJREFUCNdjYGBg+H+AwUGBwV+BQUGAQX0CiNQQYFABk8ogLgBsYQUt2gNKPwAAAABJRU5ErkJggg==);
+      .testcaselogbtn::before {
+        content: "🪵"
```

Comment:
Maybe 🖨️ or 🧾

Reply:
filter worked following [this](https://css-tricks.com/apply-a-filter-to-a-background-image/)